CMakeLists.txt

```diff
@@ -124,6 +124,10 @@ INCLUDE_DIRECTORIES (
 	${CMAKE_CURRENT_BINARY_DIR}/src/msgpack/
 )
 
+IF(MSVC)
+    SET_SOURCE_FILES_PROPERTIES(src/unpack.c src/objectc.c src/version.c src/vrefbuffer.c src/zone.c PROPERTIES LANGUAGE CXX )
+ENDIF()
+
 ADD_LIBRARY (msgpack SHARED
```

Comment:
At that time, it was reasonable change. But now, C source files are compiled as the programming language C. So, the fix above is no longer needed.